tox.ini

```diff
@@ -1,24 +1,24 @@
 [tox]
-envlist = docs,pep8,py34,py35
+envlist = docs,pep8,py38,py39
 
 [testenv]
 deps =
-    coverage
+    coverage==4.5.4
```

Comment:
Is there a specific reason why we are version locking coverage.

Reply:
Hmm... There used to be an error, but it seems to be gone now... 🤔